examples/green/05_network.yaml

```diff
@@ -1,12 +0,0 @@
-name: "Default network configuration"
-stages:
-   initramfs:
-     - name: "Setup network"
-       files:
-       - path: /etc/sysconfig/network/ifcfg-eth0
```

Comment:
Good point. If we are using NetworkManager by default, in case no network configuration is available, it will do the "right thing":create a default DHCP connection over available interfaces.